languages/csharp/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,93 @@
+Up to now, we saw how to declare variables and how to assign them to
```

Comment:
There is still quite some text in the introduction. Could you perhaps see if you can make things a bit more concise? If you'd like, I can totally help with that.